libbeat/docs/processors-using.asciidoc

```diff
@@ -157,6 +158,23 @@ range:
 ------
 
 
+[float]
+[[condition-has_fields]]
+===== `has_fields`
+
+The `has_fields` condition checks if the given fields exist in the
```

Comment:
I think this would be more clear if it said "checks if _all_ the given fields exist".